### PR TITLE
V2

### PR DESCRIPTION
**Pull request by kasimrehman**

### CODE REVIEW

File: apps/storage-adapter-mongodb/package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   "author": "",
   "license": "SEE LICENSE IN LICENSE",
   "dependencies": {
-    "i40-aas-objects": "0.0.1-alpha",
+    "i40-aas-objects": "0.2.1",
```
Comment (Manu3756) [inline apps/storage-adapter-mongodb/package.json]: 0.2.6 is the latest Verison.

**Comment by Manu3756** [comment]
@kasimrehman is this tested through a full run?

**Comment by kasimrehman** [comment]
yes

**Comment by Manu3756** [comment]
ok, please update the version of i40-aas-objects then I can merge.

**Comment by kasimrehman** [comment]
The tests need to be reviewed as they were missing from the pull request the first time round.

**Comment by akiskips** [comment]
There is an issue with the endpoint-egress regarding calling the endpoint-reg. This was resolved in the [https://github.com/SAP/i40-aas/pull/57/commits/099c2aaa5fb6757ae3e2760934364f40c139b254](commit) in the Update adapter registry Branch

**Pull request merged**